azure/packages/azure-service-utils/api-report/azure-service-utils.legacy.public.api.md

```diff
@@ -6,6 +6,4 @@
 
 export { IUser }
 
-export { ScopeType }
```

Comment:
Change seems correct...? The type lives in `@fluidframework/driver-definitions` and it's `@legacy + @alpha` (separate API report, where it still shows up).